react-components/src/context/UserContext.js

```diff
@@ -144,6 +147,10 @@ const UserContextProvider = props => {
         }
     };
 
+    const resetCustomerCart = async () => {
+        await resetCustomerCartAction({ fetchCustomerCartQuery, dispatch });
```

Comment:
Not sure if we should proxy this call. The components can bring their own query (in case of customizations) so maybe we should get that as an argument.

Reply:
I added the query as a parameter to the exposed function.